src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/model/ResynchedAlert.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.hmppsalertsapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import java.util.UUID
+
+data class ResynchedAlert(
```

Comment:
Minor: perhaps use ResyncedAlert? https://grammarwiki.com/past-tense-of-sync/#Synced_Or_Synched